src/components/app/app.tsx

```diff
@@ -18,7 +18,7 @@ const createRouter = ({ offers }: AppScreenProps) => createBrowserRouter([
     children: [
       {
         index: true,
-        element: <MainPage city={'Amsterdam'}offers={offers} />
+        element: <MainPage city={'Brussels'}offers={offers} />
```

Comment:
Пробелы не забывай между пропсами
```suggestion
        element: <MainPage city="Brussels" offers={offers} />
```